package/MDAnalysis/analysis/__init__.py

```diff
@@ -22,11 +22,26 @@
 #
 
 """
-:mod:`MDAnalysis.analysis` --- Analysis code based on MDAnalysis
+Analysis code based on MDAnalysis --- :mod:`MDAnalysis.analysis`
 ================================================================
 
 The :mod:`MDAnalysis.analysis` sub-package contains various recipes and
 algorithms that can be used to analyze MD trajectories.
+An analysis using the available modules
+usually follows the same structure
+
+#. Import the desired module
+#. Initialize the module previously imported.
+#. Run the analysis, optionally for specific trajectory slices
+#. Access the analysis from the `results` attribute
+
+ .. code-block:: python
+
+    from MDAnalysis.analysis import ExampleAnalysisModule  # (e.g. RMSD)
+
+    analysis_obj = ExampleAnalysisModule(universe, ...)
+    analysis_obj.run(start_frame, stop_frame, step)
+    print(analysis_obj.results)
 
 If you use them please check if the documentation mentions any specific caveats
 and also if there are any published papers associated with these algorithms.
```

Comment:
Please remove the whole section *Available analysis modules* — this list is not maintained anymore and it's confusing when it shows up in the docs.